about.html

```diff
@@ -0,0 +1,140 @@
+      <section id="about">
+        <p class="quick-link">
+          <a class="top-link" href="index.html">Home</a><span> &#62; </span
+          ><a class="top-link" href="about.html">About</a>
+        </p>
+        <div class="about-me">
+          <h2>"hello! Sharing world"</h2>
+          <h1>Creative commons global summit 2022</h1>
+          <p class="bio">
+            the global summit brings together thecommunity of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country everyt two years. <br />
+            This year, we hope to expand our invitation list including
+            organizations and individuals who want to work with us on shared
+            projects that advance the cause of the commons, free culture and
+            open knowledge. <br />
+            So if you're active and engaged in the worlds of open content and
+            knowledge -- free software advocates, Wikipedians, Open Knowledge,
+            galleries, libraries, museums, archives, governements and
+            foundations, lawyers, and activists -- we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+          <p class="ad-det">
+            Please contact us per email for any further question <br />
+            about CC global Summit 2015!
+          </p>
+          <p class="about-email">ccsummit2015@cckorea.org</p>
+        </div>
+      </section>
+      <section id="co-logo">
+        <h3>Creative Commons Global summit 2022 Logo</h3>
+        <hr />
+        <p class="mobile">
+          The Logo of CC Global Summit 2015 was decided <br />
+          through the logo competition from 8.June to 7.July
+        </p>
+        <p class="desktop">
+          The logo fo Company was decided through the logo competition from 8.
+          June to 7. July after online public voting and selection Committee's
+          judging process, the logo from naresh Agrawal won the prize. <br />
+          After consultation with Mr.Agrawal we optimized the prize-winnin-logo
+          for this year's summit as following.
+        </p>
+        <img src="./assets/logos/Logo.png" alt="compnay logo" />
+      </section>
+      <section id="events-container">
+        <h3>See the past CC Global Summits</h3>
+        <hr />
+        <p>
+          Take a look at the last two CC global Summits which took place in
+          Buenos Aires and in Warsaw.
+        </p>
+        <div class="events">
+          <div class="event1">
+            <h3 class="event-title">2013</h3>
+            <p class="event-details">CC global Summit 2013 in Aires</p>
+          </div>
+          <div class="event2">
+            <h3 class="event-title">2011</h3>
+            <p class="event-details">CC global Summit 2011 in Warsaw</p>
+          </div>
+        </div>
+      </section>
+      <footer id="footer" class="balck">
```

Comment:
Please ensure that the `About Page` has a dark background.

Reply:
yes, you right
I did solve  the remining part